frontend/src/scenes/cohorts/CohortFilters/CohortField.tsx

```diff
@@ -167,6 +173,118 @@ export function CohortPersonPropertiesValuesField({
     )
 }
 
+export function CohortEventFiltersField({
+    fieldKey,
+    criteria,
+    cohortFilterLogicKey,
+    onChange: _onChange,
+    groupIndex,
+    index,
+}: CohortEventFiltersFieldProps): JSX.Element {
+    const { logic } = useCohortFieldLogic({
+        fieldKey,
+        criteria,
+        cohortFilterLogicKey,
+        onChange: _onChange,
+    })
+    const { value } = useValues(logic)
+    const { onChange } = useActions(logic)
+    const componentRef = useRef<HTMLDivElement>(null)
+
+    const valueExists = ((value as AnyPropertyFilter[]) || []).length > 0
+
+    useEffect(() => {
+        // :TRICKY: We check paremt has CohortCriteriaRow__Criteria__Field class and add basis-full class if value exists
```

Comment:
```suggestion
        // :TRICKY: We check parent has CohortCriteriaRow__Criteria__Field class and add basis-full class if value exists
```

Indeed, this is pretty hacky though.